### PR TITLE
Introducing `Check it out` component to the new landing page

### PR DESCRIPTION
**Pull request by kwiatekus**
**Description**

Changes proposed in this pull request:

- Introduce new component

Screen Shot 2021-05-28 at 14 47 34

**Related issue(s)**
Fixes https://github.com/kyma-project/website/issues/687

### CODE REVIEW

File: content/i18n/en/landing-page.json [hunk 1]
```diff
@@ -98,5 +98,14 @@
     "youtubeButton": "Watch our videos",
     "lastBlogPostHeader": "Kyma blog",
     "lastBlogPostButton": "Read our posts"
+  },
+  "checkItOut": {
+    "headline": "Check It Out",
+    "firstParagraphTitle": "Install Kyma locally",
+    "firstParagraph1": "Get Kyma CLI first. Choose from convenient installation {options}.",
+    "firstParagraph2": "With Kyma CLI you are only two steps away from having the local kyma installed.",
```
Comment (majakurcius) [inline content/i18n/en/landing-page.json:106]:
```suggestion
    "firstParagraph2": "With Kyma CLI, you are only two steps away from having the local Kyma installed.",
```

File: content/i18n/en/landing-page.json [hunk 2]
```diff
@@ -98,5 +98,14 @@
...
+    "secondParagraphTitle": "Start Exploring",
+    "secondParagraph1": "Try out getting started guides that will help you explore Kyma features.",
```
Comment (majakurcius) [inline content/i18n/en/landing-page.json:108]:
```suggestion
    "secondParagraph1": "Try our getting started guides that will help you explore Kyma features.",
```

File: content/i18n/en/landing-page.json [hunk 3]
```diff
@@ -98,5 +98,14 @@
     "youtubeButton": "Watch our videos",
     "lastBlogPostHeader": "Kyma blog",
     "lastBlogPostButton": "Read our posts"
+  },
+  "checkItOut": {
+    "headline": "Check It Out",
```
Comment (klaudiagrz) [inline content/i18n/en/landing-page.json:103]:
```suggestion
    "headline": "Check it out",
```

Comment (klaudiagrz) [reply content/i18n/en/landing-page.json:103]: It would be nice if we were consistent with our approach - in some headings we capitalize every word, and in others we capitalize only the first one. Let's unify that 😉

File: content/i18n/en/landing-page.json [hunk 4]
```diff
@@ -98,5 +98,14 @@
...
+    "firstParagraphTitle": "Install Kyma locally",
+    "firstParagraph1": "Get Kyma CLI first. Choose from convenient installation {options}.",
+    "firstParagraph2": "With Kyma CLI you are only two steps away from having the local kyma installed.",
+    "secondParagraphTitle": "Start Exploring",
```
Comment (klaudiagrz) [inline content/i18n/en/landing-page.json:107]:
```suggestion
    "secondParagraphTitle": "Start exploring",
```

File: content/i18n/en/landing-page.json (hunk 2 above)
Comment (klaudiagrz) [reply content/i18n/en/landing-page.json:108]: I'd rephrase:
`Go through the Getting Started guides that will help you explore Kyma features.`

Comment (majakurcius) [reply content/i18n/en/landing-page.json]: I'd say it depends whether they are optional or not. If they're optional, I'd stil to "try out", if they must be done, I'd use "go through". Either way, I also agree we should capitalize it.

Comment (klaudiagrz) [reply content/i18n/en/landing-page.json]: I don't know how `try out` expresses "optionality" more than `go through`. In the end, it's up for the user anyway, since we cannot force anyone to go through/ try out the guides. It's just our recommendation. I'd say it's just the preference 😉 Anyway, I agree that we should capitalize `Getting Started`. I'm also for preceding it with the definite article.

```Try out the Getting Started guides that will help you explore Kyma features.```

or

```Go through the Getting Started guides that will help you explore Kyma features.```

Comment (kwiatekus) [reply content/i18n/en/landing-page.json]: Yes.. it could be totally re-phrased..
How about going in the direction :
`Explore Kyma features`
`We prepared guided steps that will help you  getting started with Kyma and discovering own use cases`

Comment (klaudiagrz) [reply content/i18n/en/landing-page.json]: Summoning @NHingerl - any opinion in that matter? 🤔

Comment (NHingerl) [reply content/i18n/en/landing-page.json]: Thou hast called and I shall answer!

I think we could even go with `Explore Kyma` without the word "features". If we choose to add "features", I'd prefer `Explore _the_ Kyma features` with an article.

Same about `own use cases`, I'd propose `_your_ own use cases`.

Also, we could pick up the headline and say `_Check out_ our [Getting Started](link) guides that will help you get familiar with Kyma and discover...`

File: src/views/landingPage/CheckItOut/Termynal/Termynal.tsx [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import React, { FC } from "react";
+import { TermynalStyles } from "./styled";
+
+export const Termynal: FC = () => (
+  <TermynalStyles>
+    <div id="termynal" data-termynal={true}>
+      <span data-ty="input">kyma provision k3s</span>
+      <span data-ty="input">kyma deploy</span>
```
Comment (NHingerl) [inline src/views/landingPage/CheckItOut/Termynal/Termynal.tsx:8]: are we sure we want to recommend the "pure" `deploy` command without eval profile? Will it work well for local installations?

File: content/i18n/en/landing-page.json (hunk 2 above)
Comment (klaudiagrz) [reply content/i18n/en/landing-page.json]: @kwiatekus you have some versions to choose from. Whichever you choose, I guess we all agree that the Getting Started part should be capitalized and preceded with either "the" or "our".

Comment (majakurcius) [reply content/i18n/en/landing-page.json]: I'm fine with your suggestions, so let's just agree on one so @kwiatekus can continue with this PR :D

File: src/views/landingPage/CheckItOut/Termynal/Termynal.tsx (hunk 5 above)
Comment (kwiatekus) [reply src/views/landingPage/CheckItOut/Termynal/Termynal.tsx:8]: The default profile works fine locally. We dont need to suggest evaluation profile on the landing page in my oprinion

File: content/i18n/en/landing-page.json (hunk 2 above)
Comment (kwiatekus) [reply content/i18n/en/landing-page.json]: Thanks. I applied your suggestions

Screen Shot 2021-05-31 at 13 48 38

**Review by klaudiagrz — APPROVED** [review]

File: src/views/landingPage/CheckItOut/Termynal/Termynal.tsx (hunk 5 above)
Comment (NHingerl) [reply src/views/landingPage/CheckItOut/Termynal/Termynal.tsx:8]: okay cool, thank you!

**Review by rJankowski93 — APPROVED** [review]
